algebird-core/src/main/scala/com/twitter/algebird/Eventually.scala

```diff
@@ -199,7 +199,7 @@ trait AbstractEventuallyAggregator[A, E, O, C] extends Aggregator[A, Either[E, O
 trait EventuallyAggregator[A, E, O, C] extends AbstractEventuallyAggregator[A, E, O, C] {
 
   //avoid init order issues and cyclical references
-  @transient override lazy val semigroup =
+  @transient override lazy val semigroup: EventuallySemigroup[E, O] =
```

Comment:
can we loosen this to `Semigroup[Either[E, O]]`?

Reply:
yup yup